src/snowflake/cli/plugins/snowpark/commands.py

```diff
@@ -323,12 +336,13 @@ def build(
     The archive is built using only the `src` directory specified in the project file.
     """
     snowpark = cli_context.project_definition
-    source = Path(snowpark.src)
-    artifact_file = _get_snowpark_artifact_path(snowpark)
-    log.info("Building package using sources from: %s", source.resolve())
+    project_root = cli_context.project_root
+    absolute_source_path = _get_snowpark_source_absolute_path(project_root, snowpark)
```

Comment:
```suggestion
    source_path = _get_snowpark_source_absolute_path(project_root, snowpark)
```
Not sure if we need this additional prefix

Reply:
N/A now